packages/ui/exports/form-core.js

```diff
@@ -49,5 +49,6 @@ export { LionValidationFeedback } from '../components/form-core/src/validate/Lio
 
 export { ChoiceGroupMixin } from '../components/form-core/src/choice-group/ChoiceGroupMixin.js';
 export { ChoiceInputMixin } from '../components/form-core/src/choice-group/ChoiceInputMixin.js';
+export { CustomChoiceGroupMixin } from '../components/form-core/src/choice-group/CustomChoiceGroupMixin.js';
```

Comment:
```suggestion
```
For now, we don't want to expose the mixin yet (as it probably will be merged into ChoiceGroupMixin)